Database/showroomProxy.js

```diff
@@ -873,7 +873,9 @@ function getStudentProject (userID, projectID, callback) { //TODO: test
 function postMeetHistory (userID, meetingID, callback) {
     logCtx.fn = 'postMeetHistory';
     var query = "insert into meethistory (projectid, userid, jointime) values ($1, $2, $3)";
-    var values = [meetingID, userID, new Date(Date.now()).toISOString()]; //TODO: current time seems to be in different time zone? it's 4 hours ahead
+    var today = new Date();
+    today.setTime(today.getTime() - 14400000); //Subtract 4 hours (in ms) to account for UTC timezone [needed for production server in ECE]
```

Comment:
Add the number to config and make it depend on the production variable for compatibility with development